content/TrueCommand/Administration/LDAPServers.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "LDAP Servers"
+weight: 40
+---
+
+{{< toc >}}
+
+Users can configure TrueCommand to use LDAP servers for security and authentication management among connected TruNAS systems.
+
+## Add an LDAP Server in TrueCommand
+
+Click the gear icon in the upper toolbar and select **Administration**. Scroll down to the **LDAP Servers** widget and click **ADD**.
+
+![AddLDAPServer](/images/TrueCommand/AddLDAPServer.png "Add LDAP Servers")
+
+| Field | Description |
+|---------|-------------|
+| **Hostname** | Hostname of LDAP/AD server, with optional port. e.g. example.com:636. |
+| **Domain** | LDAP base domain. e.g. dc=example,dc=com. |
+| **Group Domain** | Optional for admins that want to avoid issues with unwanted groups by forcing group searches to start at a deeper domain. |
+| **User ID Field** | Class mapped to login username. The default is uid. |
```

Comment:
host name, user name, file system, are all two words unless a cli command or taking about the UI element when it is a combined word.
```suggestion
| **User ID Field** | Class mapped to login user name. The default is uid. |
```